README.md

```diff
@@ -7,44 +7,14 @@
 
 ## PEL parser
 
-The pel.peltool.peltool module parses PEL files and prints the resulting PEL
-fields in JSON.  It will load and run the various component parsers if they are
-available.
+- Peltool parser is a Python module that takes binary PEL data as input and displays the output in JSON format.
+- Peltool parser can be used on both BMC and non-BMC environments.
```

Comment:
I feel, "It will load and run the various component parsers if they are available" need to capture to mention about other firmware parser modules.

Reply:
please help me understand this

Reply:
I feel we no need to remove this statement. Just add the same.